packages/bunnymen-core/src/node.ts

```diff
@@ -240,17 +240,21 @@ export class Node extends EventEmitter {
       transports.push(tcp())
     }
     return createLibp2p({
+      peerId: this._opts.peerId,
       addresses: {
         // Add the signaling server address, along with our PeerId to our multiaddrs list
         // libp2p will automatically attempt to dial to the signaling server so that it can
         // receive inbound connections from other peers
         listen: [
           //'/dns4/wrtc-star1.par.dwebops.pub/tcp/443/wss/p2p-webrtc-star',
           //'/dns4/wrtc-star2.sjc.dwebops.pub/tcp/443/wss/p2p-webrtc-star',
+          '/dns4/bunnymen.delvelabs.xyz/tcp/443/wss/p2p-webrtc-star/',
```

Comment:
🔥

Reply:
oh ya. i forgot that i added this too. i will add it to the PR summary